core/server/middleware/error-handler.js

```diff
@@ -4,9 +4,11 @@ var _ = require('lodash'),
     config = require('../config'),
     errors = require('../errors'),
     i18n = require('../i18n'),
-    _private = {};
+    _private = {},
+    errorHandler = {},
+    htmlStackExclusions = ['NotFoundError', 'DatabaseNotPopulatedError'];
 
-_private.parseStack = function (stack) {
+_private.parseStack = function parseStack(stack) {
```

Comment:
ok agree. maybe raise some questions in DM to me. 

but one change i would like to see: can we check error instances?
```if (!err instanceof errors.NotFoundError && !err instanceof DatabaseNotPopulatedError)```